llvm/test/CodeGen/NVPTX/vector-returns.ll

```diff
@@ -0,0 +1,523 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_20 | FileCheck %s
+
+; XFAIL: *
+; FIXME: Fails on `byte2` because NVPTX wants to lower v2i8 to 2 i8s while 
+;   SelectionDAG wants to lower it to 1 v2i16. 
```

Comment:
I'm not quite sure what `XFAIL: *` would do here. Will we ignore the failure if some other tests will start failing?

I'd just comment out the failing test with a TODO, and still check the rest.

Reply:
> I'm not quite sure what XFAIL: * would do here.
It will test that the test always fails. 

> Will we ignore the failure if some other tests will start failing?
Correct.

> I'd just comment out the failing test with a TODO, and still check the rest.
Sounds good. Done in 3cea7d62363b650852a685ddb4316680ee8a77a9.